src/site/content/en/learn/css/inheritance/index.md

```diff
@@ -0,0 +1,341 @@
+- [border-spacing](https://developer.mozilla.org/en-US/docs/Web/CSS/border-spacing)
+- [caption-side](https://developer.mozilla.org/en-US/docs/Web/CSS/caption-side)
+- [color](https://developer.mozilla.org/en-US/docs/Web/CSS/color)
+- [cursor](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor)
+- [direction](https://developer.mozilla.org/en-US/docs/Web/CSS/direction)
+- [empty-cells](https://developer.mozilla.org/en-US/docs/Web/CSS/empty-cells)
+- [font-family](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family)
+- [font-size](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size)
+- [font-style](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style)
+- [font-variant](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant)
+- [font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)
+- [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font)
+- [letter-spacing](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing)
+- [line-height](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height)
+- [list-style-image](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-image)
+- [list-style-position](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position)
+- [list-style-type](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type)
+- [list-style](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style)
+- [orphans](https://developer.mozilla.org/en-US/docs/Web/CSS/orphans)
+- [quotes](https://developer.mozilla.org/en-US/docs/Web/CSS/quotes)
+- [text-align](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align)
+- [text-indent](https://developer.mozilla.org/en-US/docs/Web/CSS/text-indent)
+- [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)
+- [visibility](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility)
+- [white-space](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space)
+- [widows](https://developer.mozilla.org/en-US/docs/Web/CSS/widows)
+- [word-spacing](https://developer.mozilla.org/en-US/docs/Web/CSS/word-spacing)
+
+## How inheritance works
+
+Every HTML element has every CSS property defined by default with an initial value.
+An initial value is a property that's not inherited and shows up as a default
+if the cascade fails to calculate a value for that element.
+
+<figure class="w-figure">
+{% Video src="video/VbAJIREinuYvovrBzzvEyZOpw5w1/OvoYqOMcdFZL7wJQIL2C.mp4" %}
+</figure>
+
+Properties that can be inherited cascade downwards,
+and child elements will get a computed value which represents its parent's value.
+This means that if a parent has `font-weight` set to `bold` all child elements will be bold,
+unless their `font-weight` is set to a different value,
+or the user agent stylesheet has a value for `font-weight` for that element.
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
```

Comment:
In this demo, can you call out that in the `span` you're *overriding* the bold value? 

"This sentence is wrapped in a <span>, so the text is normal weight." --> "This sentence is wrapped in a <span>, where we are overriding the bold value to `font-weight: normal`, so the text is normal weight."

Reply:
Might want to flip this so the font-weight is light on everything and bolder on the span tag ( a more likely scenario)

Reply:
Yeh that's a good shout